### PR TITLE
Added EE countries for cmp

### PR DESCRIPTION
**Pull request by fabianScout**

### CODE REVIEW

File: src/cashstack.js [hunk 1]
```diff
@@ -386,6 +397,16 @@ function loadGTM() {
         it: 'GTM-WTCSNR',
         nl: 'GTM-TW48BJ',
         com: 'GTM-KWX9NX',
+        bg: 'GTM-KWX9NX',
```
Comment (bdadam) [inline src/cashstack.js:400]: We already have com as fallback:
```js
const containerId = containerIdsByTld[tld] || containerIdsByTld['com'];
```

File: dist/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
     (function () {
         'use strict';
 
-        var releasedCountries = ['at', 'be', 'com', 'de', 'es', 'fr', 'nl', 'lu', 'it'];
+        var releasedCountries = ['at', 'be', 'com', 'de', 'es', 'fr', 'nl', 'lu', 'it', 'bg', 'cz', 'ru', 'pl', 'hu', 'hr', 'ro', 'se', 'tr', 'ua'];
```
Comment (bdadam) [inline dist/index.html:5]: No need for this line and the if statement below anymore. With this PR the CMP will be released on all domains.

File: dist/index.html [hunk 3]
```diff
@@ -186,7 +196,7 @@
 
             function isInCmpExperiment() {
                 try {
-                    var releasedCountries = ['at', 'be', 'com', 'de', 'es', 'fr', 'nl', 'lu', 'it'];
+                    var releasedCountries = ['at', 'be', 'com', 'de', 'es', 'fr', 'nl', 'lu', 'it', 'bg', 'cz', 'ru', 'pl', 'hu', 'hr', 'ro', 'se', 'tr', 'ua'];
```
Comment (bdadam) [inline dist/index.html:199]: Same as above, no need to check for `releasedCountries` anymore.

File: src/cashstack.js [hunk 4]
```diff
@@ -1,5 +1,6 @@
 (function () {
-    var releasedCountries = ['at', 'be', 'com', 'de', 'es', 'fr', 'nl', 'lu', 'it'];
+    var releasedCountries = ['at', 'be', 'com', 'de', 'es', 'fr', 'nl', 'lu', 'it', 'bg', 'cz', 'ru', 'pl', 'hu', 'hr', 'ro', 'se', 'tr', 'ua'];
```
Comment (bdadam) [inline src/cashstack.js:2]: Same as above.

**Review by bdadam — CHANGES_REQUESTED** [review]

File: dist/index.html (hunk 2 above)
Comment (fabianScout) [reply dist/index.html]: fixed

**Review by bdadam — APPROVED** [review]

**Pull request merged**